crates/binstalk/src/ops/resolve.rs

```diff
@@ -316,6 +318,14 @@ async fn download_extract_and_verify(
         .iter()
         .zip(bin_files)
         .filter_map(|(bin, bin_file)| {
+            // skip binaries that were not requested by user
+            if bins
+                .as_ref()
+                .is_some_and(|bins| !bins.contains(&bin_file.base_name))
```

Comment:
```suggestion
                .is_some_and(|bins| !bins.contains(bin.name.as_str()))
```


`base_name` includes the `.exe` suffix